git_machete/__init__.py

```diff
@@ -1 +1,5 @@
+import os
+
 __version__ = '3.4.0'
+
+ROOT_DIR = os.path.abspath(__file__).replace('git_machete/__init__.py', '')
```

Comment:
I would suggest not to keep this file in repository, cause it would have to be added to `gitignore` in every case and if someone accidentally push this file to github then token gets immediately invalid.  
As suggested in issue #233, i think it's better to keep it in users directory of `~/` which makes token available for every repository that user may have. 

if it has to be kept in repository for some reason, maybe put this file next to definition file under `.git` directory?